power_grid_model_c/power_grid_model/include/power_grid_model/power_grid_model.hpp

```diff
@@ -45,19 +46,20 @@ using IdxCount = boost::counting_iterator<Idx>;
 struct Idx2D {
     Idx group;  // sequence number of outer module/groups
     Idx pos;    //  sequence number inside the group
+
+    friend constexpr bool operator==(Idx2D x, Idx2D y) = default;
 };
-inline bool operator==(Idx2D x, Idx2D y) {
-    return x.group == y.group && x.pos == y.pos;
-}
 
 // math constant
 using namespace std::complex_literals;
 using DoubleComplex = std::complex<double>;
-constexpr double sqrt3 = 1.73205080756887729;
-constexpr double sqrt3_inv = 1.0 / sqrt3;
+
+using std::numbers::inv_sqrt3;
+using std::numbers::pi;
+using std::numbers::sqrt3;
```

Comment:
for some reason is sqrt3 popular and taken in the standard :)

Reply:
ikr hahaha same for `inv_sqrt3` 😄